### PR TITLE
add unsubmit button for students for submittable and submitted levels

### PR DESCRIPTION
**Pull request by laurelfan**
- unsubmit
  - button for students: 
  - that pops up this dialog: 
- show submitted on timestamp in teacher's local time instead of UTC: 
- change Submit button to purple 
- unsubmit turns the circle yellow (attempted) instead of white (never attempted)

### CODE REVIEW

File: dashboard/app/views/levels/_teacher.html.haml [hunk 1]
```diff
@@ -28,15 +28,20 @@
...
-                  %h4= t('teacher.panel.submitted_on', date: @user_level.updated_at)
+                #submitted
+                  %div= t('teacher.panel.submitted_on')
+                  .timestamp= @user_level.updated_at
                   %button.btn.btn-info#unsubmit{"data-user-level-url" => url_for(@user_level)}
                     = t('teacher.panel.unsubmit')
                   :javascript
+                    $(function() {
+                      var submittedTimestamp = $('#submitted .timestamp');
+                      submittedTimestamp.text((new Date(submittedTimestamp.text())).toLocaleString());
```
Comment (joshlory) [inline dashboard/app/views/levels/_teacher.html.haml:39]: I don't think `.toLocaleString()` is available below IE11.

Comment (laurelfan) [inline dashboard/app/views/levels/_teacher.html.haml:39]: I tried it on IE9 on saucelabs and it does have toLocaleString.  According to do the doc IE<11 doesn't support toLocaleString() with the locale as an argument. The doc was kind of unclear about whether it supports it without an argument as I'm using it here but I tried it and it seems to:



However, IE9 doesn't like to parse the output of Time#to_s, fixing...

Comment (joshlory) [inline dashboard/app/views/levels/_teacher.html.haml:39]: Ah, cool!  I'm totally going to start using this.

Comment (laurelfan) [inline dashboard/app/views/levels/_teacher.html.haml:39]: want to review the rest of this while you are here? :)

**Comment by joshlory** [comment]
LGTM after resolving merge conflicts.

**Pull request merged**